package.json

```diff
@@ -1,80 +1,85 @@
     "lint-fix": "npm run lint -- --fix",
-    "hot-server": "cross-env NODE_ENV=development node --max_old_space_size=2096 -r babel-register server.js",
     "build-main": "yarn build-main-quiet --progress --profile --colors",
-    "build-main-quiet": "cross-env NODE_ENV=production node -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.electron.js",
+    "build-main-quiet": "cross-env NODE_ENV=production webpack --config webpack.config.main.prod.babel.js",
     "build-renderer": "yarn build-renderer-quiet --progress --profile --colors",
-    "build-renderer-quiet": "cross-env NODE_ENV=production node -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.production.js",
-    "build": "npm run build-main && npm run build-renderer",
-    "build-quiet": "npm run build-main-quiet && npm run build-renderer-quiet",
-    "start": "cross-env NODE_ENV=production electron ./app/",
-    "start-hot": "cross-env HOT=1 NODE_ENV=development electron -r babel-register -r babel-polyfill ./app/main.development",
+    "build-renderer-quiet": "cross-env NODE_ENV=production webpack --config webpack.config.renderer.prod.babel.js",
+    "build": "concurrently \"yarn build-main\" \"yarn build-renderer\"",
+    "build-quiet": "concurrently \"yarn build-main-quiet\" \"yarn build-renderer-quiet\"",
+    "start": "cross-env NODE_ENV=production electron ./app/main.prod.js",
     "postinstall": "concurrently \"electron-builder install-app-deps\" \"node node_modules/fbjs-scripts/node/check-dev-engines.js package.json\"",
-    "dev": "npm run hot-server -- --start-hot",
-    "prepare-qa-build": "node -r babel-register scripts/prepare-qa-build.js",
+    "dev": "cross-env START_HOT=1 yarn start-renderer-dev",
+    "start-renderer-dev": "cross-env NODE_ENV=development webpack-dev-server --config webpack.config.renderer.dev.babel.js",
+    "start-main-dev": "cross-env HOT=1 NODE_ENV=development electron -r @babel/register ./app/main.dev.js",
+    "prepare-qa-build": "node -r @babel/register scripts/prepare-qa-build.js",
     "package": "npm run build-quiet && electron-builder -p always",
     "package-win": "npm run build && electron-builder --win --x64",
     "package-mac": "npm run build && electron-builder --mac",
     "package-linux": "npm run build && electron-builder --linux",
     "package-all": "npm run build && electron-builder -mwl"
   },
   "dependencies": {
+    "@hot-loader/react-dom": "16.11.0",
     "@material-ui/core": "3.9.3",
     "@material-ui/icons": "3.0.2",
     "async": "2.6.1",
     "babyparse": "0.4.6",
     "ble-glucose": "0.2.2",
-    "bows": "1.7.0",
+    "bows": "1.7.2",
     "chrome-launcher": "0.12.0",
     "classnames": "2.2.6",
     "commander": "4.0.1",
+    "core-js": "2.6.10",
     "cp2102": "0.0.3",
     "decompress": "4.2.0",
-    "electron-debug": "2.2.0",
+    "electron-debug": "3.0.1",
     "electron-is-dev": "1.1.0",
+    "electron-log": "3.0.9",
+    "electron-updater": "4.2.0",
```

Comment:
Why did this move from `devDependencies` to `dependencies`?

Reply:
Because it's one of the packages that is required _after_ the build process directly by `main.dev.js` (and this is where it lives in the boilerplate project). It may also have to do with the fact that the boilerplate uses the Webpack DLL splitting feature for all of the things under `dependencies` to help with build time (mostly repeat-build time I believe) and doesn't use the two `package.json` layout anymore.